### DISCUSSION TITLE
TensorFlow JS

### DISCUSSION THREAD

**Discussion by H4ckR0B0t** [mauricio2093/tensorflowJS#4] [Q&A] [answered]
Donde puedo encontrar los tutoriales de IBM de tensorflow ?

**Comment by mauricio2093** [comment, marked as answer]
aquí lo puedes ver https://developer.ibm.com/tutorials/coding-a-deep-learning-model-using-tensorflow-javascript/?cm_mmc=OSocial_Blog-_-Developer_IBM+Developer-_-WW_WW-_-ibmdev-OInfluencer-YouTube-KA-tensorflow-javascript&cm_mmca1=000037FD&cm_mmca2=10010797
